### PR TITLE
Endpoints for the remaining actor methods.

### PR DESCRIPTION
**Pull request by FirestarJes**
There are still a matter of what we want to update in an actor, so this will probably need updating later.

### CODE REVIEW

File: source/Energinet.DataHub.MarketParticipant.EntryPoint.Organization/Functions/UpdateActorFunction.cs [hunk 1]
```diff
@@ -67,8 +67,15 @@ private static async Task<UpdateActorCommand> UpdateActorCommandAsync(HttpReques
...
+                var gln = query.Get("gln") ?? string.Empty;
+                var status = query.Get("status") ?? string.Empty;
 
-                return new UpdateActorCommand(organizationId, actorId, marketRoles);
+                if (!Guid.TryParse(organizationId, out var orgGuid))
+                    throw new ValidationException("Invalid organizationId, must be a valid GUID");
+                if (!Guid.TryParse(actorId, out var actorGuid))
+                    throw new ValidationException("Invalid actorId, must be a valid GUID");
+
+                return new UpdateActorCommand(orgGuid, actorGuid, new ChangeActorDto(new GlobalLocationNumberDto(gln), status,  marketRoles));
```
Comment (michaelmajgaard) [inline source/Energinet.DataHub.MarketParticipant.EntryPoint.Organization/Functions/UpdateActorFunction.cs:78]: two white spaces after `status,`

File: source/Energinet.DataHub.MarketParticipant.EntryPoint.WebApi/Controllers/ActorController.cs [hunk 2]
```diff
@@ -13,8 +13,10 @@
 // limitations under the License.
 
 using System;
+using System.Collections.Generic;
```
Comment (michaelmajgaard) [inline source/Energinet.DataHub.MarketParticipant.EntryPoint.WebApi/Controllers/ActorController.cs:16]: unused using

Comment (FirestarJes) [reply source/Energinet.DataHub.MarketParticipant.EntryPoint.WebApi/Controllers/ActorController.cs:16]: Fixed

File: source/Energinet.DataHub.MarketParticipant.EntryPoint.Organization/Functions/UpdateActorFunction.cs (hunk 1 above)
Comment (FirestarJes) [reply source/Energinet.DataHub.MarketParticipant.EntryPoint.Organization/Functions/UpdateActorFunction.cs:78]: Fixed

File: source/Energinet.DataHub.MarketParticipant.Application/Handlers/GetActorHandler.cs [hunk 3]
```diff
@@ -56,9 +56,12 @@ public async Task<GetSingleActorResponse> Handle(GetSingleActorCommand request,
...
 
-            return actor is null
-                ? new GetSingleActorResponse(false, null)
-                : new GetSingleActorResponse(true, OrganizationMapper.Map(actor));
+            if (actor == null)
+            {
+                throw new NotFoundValidationException(request.ActorId);
+            }
+
+            return new GetSingleActorResponse(OrganizationMapper.Map(actor));
```
[the remark this answers is not in the document]
Comment (FirestarJes) [reply source/Energinet.DataHub.MarketParticipant.Application/Handlers/GetActorHandler.cs:62]: ` 
return actor switch
            {
                null => throw new NotFoundValidationException(request.ActorId),
                _ => new GetSingleActorResponse(OrganizationMapper.Map(actor))
            };
`

I've replaced it with this instead, seems better, what do you think @michaelmajgaard

**Review by defectiveAi — APPROVED** [review]

**Pull request merged**